packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -3625,6 +3726,12 @@ void main() {
     handlePos = endpoints[0].point + startHandleAdjustment;
     newHandlePos = handlePos + toNextLine;
     gesture = await tester.startGesture(handlePos, pointer: 7);
+    // Move handle up a small amount before dragging it down so the total global
+    // distance travelled can be accepted by the selection handle's [PanGestureRecognizer] as a drag.
+    // This way it can declare itself the winner before the [TapAndDragGestureRecognizer] that
+    // is on the selection overlay.
+    await tester.pump();
+    await gesture.moveTo(handlePos - toNextLine);
```

Comment:
I ran your branch locally to double check that my handle dragging stuff still works correctly and it does 👍 I can't tell a difference between now and before this PR.  So these changes to the handle dragging tests are fine with me.